### ISSUE TITLE
ScribeFire couldn't publish your post

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#328]
```
What's the problem?

ScribeFire couldn't publish your post. Here's the error message that bubbled 
up: User does not have permission to create new post

(but they all do, in google settings and blogger settings)

What browser are you using?


both Chrome and Firefox (issue same in both)


What version of ScribeFire are you running?

3.5.3.3
```

-----
Original issue reported on code.google.com by `WigWamMe...@gmail.com` on 12 May 2011 at 1:35

**Comment by GoogleCodeExporter** [comment]
```
Issue 329 has been merged into this issue.
```

-----
Original comment by `cfinke` on 22 May 2011 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
Can you remove and re-add your blog, making sure to enter your blog's URL and 
not blogspot.com or blogger.com?
```

-----
Original comment by `cfinke` on 3 Nov 2011 at 3:18
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
I have done so. Sometimes it works and sometimes it doesn't, but when it does 
it only lasts one posting and then the message pops up again.
```

-----
Original comment by `kittyfis...@gmail.com` on 3 Nov 2011 at 6:19

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 4 Nov 2011 at 2:40
* Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 4.2.1 for Chrome. Waiting on approval from Firefox.
```

-----
Original comment by `scribefi...@gmail.com` on 4 Sep 2013 at 10:02
* Changed state: **Fixed**
